README.rst

```diff
@@ -18,6 +18,30 @@ Installation
 
     conda install -c conda-forge nc_time_axis
 
+
+Example Code
+------------
+::
+
+    import nc_time_axis
+	import netcdftime
+	import matplotlib.pyplot as plt
+	import random
+	DT = [netcdftime.datetime(year=2017, month=2, day=day) for day in range(1, 31)]
+	CDT = [nc_time_axis.CalendarDateTime(item, "360_day") for item in DT]
+	TempList = []
+	for item in range(30):
+		DayTemp = round(random.uniform(0, 12), 3)
+		TempList.append(DayTemp)
+	plt.plot(CDT, TempList)
```

Comment:
Could you add an empty line above this to separate the data definition section from the plotting section.

Having extra lines makes the code more readable which is important for maintainability.

You can see some examples of this in the matplotlib gallery e.g. [this one ](https://matplotlib.org/examples/ticks_and_spines/ticklabels_demo_rotation.html)